doc/rst/language/spec/conversions.rst

```diff
@@ -12,79 +12,188 @@ expression can be a type expression. We refer to these two types the
 implicit (:ref:`Implicit_Conversions`) or
 explicit (:ref:`Explicit_Conversions`).
 
-   *Open issue*.
-
-   Should Chapel allow for user-defined conversions?
-   If so, how would the user define them?
-
 .. _Implicit_Conversions:
 
 Implicit Conversions
 --------------------
 
-An *implicit conversion* is a conversion that occurs implicitly, that
-is, not due to an explicit specification in the program. Implicit
-conversions occur at the locations in the program listed below. Each
-location determines the target type. The source and target types of an
-implicit conversion must be allowed. They determine whether and how the
-expression’s value changes.
+An *implicit conversion* is a conversion that occurs implicitly - that is -
+without an explicit specification in the program. Implicit conversions
+fall into the following categories:
+
+ * implicit conversions for initialization and assignment
+ * implicit conversions for function calls
+ * implicit conversions for conditionals
```

Comment:
So we have two dimensions here: where it occurs and between which types it is allowed. One way to structure is:

* Start with the subsections on the standard conversions: numeric, subclass, generic. In each of these, indicate whether they apply to init/assign and/or function calls.

* Then place the three sections listed above. In the first and second of them reference the appropriate standard conversions before introducing user-defined ones.